src/ledger/LedgerManagerImpl.cpp

```diff
@@ -595,11 +610,11 @@ LedgerManagerImpl::closeLedger(LedgerCloseData const& ledgerData)
     // first, prefetch source accounts fot txset, then charge fees
     prefetchTxSourceIds(txs);
     processFeesSeqNums(txs, ltx, txSet->getBaseFee(header.current()),
-                       ledgerCloseMeta);
+                       mNextMetaToEmit);
```

Comment:
see my other comment: it's better to continue to use a local variable here

Reply:
(also fixed)